libraries/js/src/payloads.ts

```diff
@@ -54,6 +58,36 @@ function parseMessage(message: string): SiwxMessage {
   };
 }
 
+// SIWA is switching away from this, but we should still support it for now
+function verifySignatureHashMaybeWrapped(publicKey: string, signature: string, message: Uint8Array): boolean {
+  const unwrappedSigned = message.length > 256 ? blake2AsU8a(message) : message;
```

Comment:
This is from how the Polkadot-SDK signs payloads. We might want to require it, but for now we support it either way.